src/scripts/extensions/authenticationHelper.ts

```diff
@@ -71,6 +72,11 @@ export class AuthenticationHelper {
 				if (isValidUser) {
 					const dataBoundaryHelper = new UserDataBoundaryHelper();
 					let userDataBoundary: string = await dataBoundaryHelper.getUserDataBoundary(response.data);
+					// The default logging has been configured to EU Pipeline. Once we find the
+					// userdataboundary and if it is different from EUDB , reinit the logger with WW Pipeline
+					if (userDataBoundary === DataBoundary[DataBoundary.GLOBAL] || userDataBoundary === DataBoundary[DataBoundary.PUBLIC]) {
```

Comment:
Also what is the flushing logic for this? Does a network request goes for every log line or is it batched and then sent?

Reply:
Flushing logic is handled in batches by Aria-Web-Telemetry logger library which captures all the logging happening into its worker thread cache and flushes periodically.